popup/popup.css

```diff
@@ -0,0 +1,145 @@
+body {
+    width: 340px;
+}
+
+/* Setting button */
+.setting-btn {
+    /* margin-top: 5px; */
+    font-size: 20px;
+    vertical-align: bottom;
+    float:right;
+    color: lightgray;
+    cursor: pointer;
+    background-color: white;
+    border: none;
+}
+.setting-btn:hover {
+    color: grey;
+}
+.setting-btn:focus {
+    outline: none;
+}
+#home-btn {
+    display: none;
+}
+/* Input field without highlight*/
+.form-control:focus {
+    box-shadow: none;
+    outline: 0px !important;
+    border: solid 1px #3bad74;
+}
+
+.plus-icon {
+    display: block;
+    height: 20px !important;
+    width: 20px !important;
+    margin-top: 14px;
+    margin-left: 3px;
+    color: gray;
+}
+
+/* Search bar */
+.search-box {
+    border: none;
+    border-bottom: solid 1px lightgrey;
+    border-radius: 0;
+}
+.search-box:focus {
+    border: none !important;
+    border-bottom: solid 1px #3bad74 !important;
+}
+
+#icon {
```

Comment:
avoid naming ids as simpletons like `icon`, `url`, etc., it's confusing and they should be reserved for other purposes.